fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -1871,6 +1871,12 @@ Future<RangeResult> AdvanceVersionImpl::getRange(ReadYourWritesTransaction* ryw,
 }
 
 ACTOR static Future<Optional<std::string>> advanceVersionCommitActor(ReadYourWritesTransaction* ryw, Version v) {
+	Optional<Standalone<StringRef>> versionEpochValue = wait(ryw->getTransaction().get(versionEpochKey));
+	if (versionEpochValue.present()) {
+		return ManagementAPIError::toJsonString(
+		    false, "advanceversion", "Illegal to modify the version while the version epoch is enabled");
+	}
```

Comment:
I wonder if there is a use case for this. For example, if we are in the process of moving data between clusters we could potentially advance the version of the target cluster by a small amount to shorten the flushing downtime. Then we would let the destination cluster drift back to clock alignment.

We aren't making use of this now, so I'm ok to disable it until we are confident that it would be useful. I just wanted to record the thought.

Reply:
This is still allowed, but would have to be done through the `versionepoch` command. Since this change reverts some of the previous implemented functionality we talked about where it was possible to do something like `targetversion add 100`, this functionality is now only available by making the version epoch larger by setting its value absolutely. Re-using the `advanceversion` command to increase the version does seem like a good use case and we can add it back in the future.

Reply:
Yeah, I think we could still accomplish this task with the `versionepoch` command, though it's a little bit more roundabout if we want to do only a temporary increase. Even so, it's possible that using `versionepoch` has better properties and would be the correct tool to use for the situation I described.